src/Kernel/Kernel.csproj

```diff
@@ -10,15 +10,15 @@
 
   <ItemGroup>
     <None Remove="res\kernel.js" />
-    <None Remove="res\logo-64x64.png" />
```

Comment:
The Jupyter kernelspec property name for the logo is automatically derived from this filename, and Jupyter relies on this being named `logo-64x64.png` in order for it to show up as the logo. It may be easiest if you just add the new logo with the same filename.

Reply:
Ah, thanks for the catch. Fixing. :)